src/test/scala/org/broadinstitute/dsde/firecloud/service/NIHWhitelistSpec.scala

```diff
@@ -0,0 +1,47 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import org.broadinstitute.dsde.firecloud.core.NIHWhitelistUtils
+import org.broadinstitute.dsde.firecloud.utils.DateUtils
+import org.scalatest.FreeSpec
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class NIHWhitelistSpec extends FreeSpec {
+
+  val usernames = Future { Map("fcSubjectId1" -> "nihUsername1", "fcSubjectId2" -> "nihUsername2") }
+
+  val expiretimes1 = Future { Map("fcSubjectId1" -> DateUtils.nowMinus24Hours.toString, "fcSubjectId2" -> DateUtils.nowPlus24Hours.toString) }
+  val currentUsernames1 = Map("fcSubjectId2" -> "nihUsername2")
+
+  val expiretimes2 = Future { Map("fcSubjectId1" -> DateUtils.nowMinus24Hours.toString) }
+  val expiretimes3 = Future { Map("fcSubjectId1" -> DateUtils.nowMinus24Hours.toString, "fcSubjectId2" -> "not a number") }
+
+  "NihWhitelistSpec" - {
+    "when handling expired and unexpired users" - {
+      "should only include unexpired users" in {
+        assertResult(currentUsernames1) {
+          Await.result(NIHWhitelistUtils.filterForCurrentUsers(usernames, expiretimes1), 1 second)
```

Comment:
These give compile warnings when running tests, pretty simple fix:

```
[warn] /Users/davidan/work/src/firecloud-orchestration/src/test/scala/org/broadinstitute/dsde/firecloud/service/NIHWhitelistSpec.scala:24: postfix operator second should be enabled
[warn] by making the implicit value scala.language.postfixOps visible.
[warn] This can be achieved by adding the import clause 'import scala.language.postfixOps'
[warn] or by setting the compiler option -language:postfixOps.
[warn] See the Scala docs for value scala.language.postfixOps for a discussion
[warn] why the feature should be explicitly enabled.
[warn]           Await.result(NIHWhitelistUtils.filterForCurrentUsers(usernames, expiretimes1), 1 second)
[warn]                                                                                            ^
[warn] /Users/davidan/work/src/firecloud-orchestration/src/test/scala/org/broadinstitute/dsde/firecloud/service/NIHWhitelistSpec.scala:32: postfix operator second should be enabled
[warn] by making the implicit value scala.language.postfixOps visible.
[warn]           Await.result(NIHWhitelistUtils.filterForCurrentUsers(usernames, expiretimes2), 1 second)
[warn]                                                                                            ^
[warn] /Users/davidan/work/src/firecloud-orchestration/src/test/scala/org/broadinstitute/dsde/firecloud/service/NIHWhitelistSpec.scala:40: postfix operator second should be enabled
[warn] by making the implicit value scala.language.postfixOps visible.
[warn]           Await.result(NIHWhitelistUtils.filterForCurrentUsers(usernames, expiretimes3), 1 second)
[warn]                                                                                            ^
[warn] three warnings found
```